datafusion/optimizer/Cargo.toml

```diff
@@ -45,6 +45,7 @@ async-trait = { workspace = true }
 chrono = { workspace = true }
 datafusion-common = { workspace = true, default-features = true }
 datafusion-expr = { workspace = true }
+datafusion-functions = { workspace = true }
```

Comment:
I think it is very important to not add this dependency -- we are triyng to make the core not know about the functions

Reply:
Cool, I can wait on it and use that function.